rancher_pr_deploy.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+DEPENDANT_SERVICE=$1
+
+create_parameter_name() {
+  echo "$1" | tr '-' '_'
```

Comment:
- is an invalid character for parameter names but not elsewhere (e.g. stack names)